### PR TITLE
Bump fwk_core and fwk_io in CI run

### PR DESCRIPTION
**Pull request by ed-xmos**
* UPDATED: To fwk_io v3.3.0 from v3.0.1 for CI run
  * UPDATED: To fwk_core v1.0.2 from v1.0.0 for CI run

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,8 @@ RTOS Framework change log
 
   * FIXED: Added dummy definitions of vTaskCoreAffinitySet() and vTaskCoreAffinitySet() functions to
     avoid warnings when configNUM_CORES is set to 1.
+  * UPDATED: To fwk_io v3.3.0 from v3.0.1 for CI run
+  * UPDATED: To fwk_core v1.0.2 from v1.0.0 for CI run
```
Comment (lucianomartin) [inline CHANGELOG.rst:10]: Do we list changes related to the tests in the changelog?

File: tools/ci/fetch_ci_deps.cmake [hunk 2]
```diff
@@ -8,7 +8,7 @@
         FetchContent_Declare(
             fwk_io
             GIT_REPOSITORY https://github.com/xmos/fwk_io.git
-            GIT_TAG        f35192c19824d60740003be5448c67f5df927014
+            GIT_TAG        dcf1a87ddd006c328906094ec12493e604bc9b2f
```
Comment (lucianomartin) [inline tools/ci/fetch_ci_deps.cmake:11]: can we use v3.3.0 instead of the hash here?

File: tools/ci/fetch_ci_deps.cmake [hunk 3]
```diff
@@ -25,7 +25,7 @@
         FetchContent_Declare(
             fwk_core
             GIT_REPOSITORY https://github.com/xmos/fwk_core.git
-            GIT_TAG        9e4f6196386995e2d7786b376091404638055639
+            GIT_TAG        c4582cb3ce4da34c8757a6f8f7df8935496038eb
```
Comment (lucianomartin) [inline tools/ci/fetch_ci_deps.cmake:28]: can we use v1.0.2 instead of the hash here?

**Review by lucianomartin — COMMENTED** [review]
Just few questions.

File: tools/ci/fetch_ci_deps.cmake (hunk 2 above)
Comment (ed-xmos) [reply tools/ci/fetch_ci_deps.cmake:11]: I am not sure. Let me try..

File: CHANGELOG.rst (hunk 1 above)
Comment (ed-xmos) [reply CHANGELOG.rst:10]: I don't see why not - I think it's useful info for the user to understand what it is verified against

**Review by lucianomartin — APPROVED** [review]
Looks good!

File: tools/ci/fetch_ci_deps.cmake (hunk 2 above)
Comment (ed-xmos) [reply tools/ci/fetch_ci_deps.cmake]: Yes this works fine and is much clearer - thanks

**Pull request merged**